integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/IngressSteps.java

```diff
@@ -26,9 +25,32 @@ public IngressSteps(TestContext context) {
         this.context = context;
     }
 
-    @Then("^send cloud events to the ingress at the endpoint with access token:$")
-    public void testIngressEndpoint(String cloudEvent) {
-        String endpoint = BridgeUtils.getOrRetrieveBridgeEndpoint(context);
+    @And("^the Ingress of Bridge \"([^\"]*)\" is deployed within (\\d+) (?:minute|minutes)$")
```

Comment:
Maybe `available` instead of `deployed`? Deployed sounds to me like we check status of some deployment or pod, in this case we check that URL is available.

Reply:
updated